jpo-ode-common/src/main/java/us/dot/its/jpo/ode/util/JsonUtils.java

```diff
@@ -57,14 +61,21 @@ public static String toJson(Object o, boolean verbose) {
    }
 
    public static Object fromJson(String s, Class<?> clazz) {
-      return gsonCompact.fromJson(s, clazz);
-//      Object o = null;
-//      try {
-//         o = mapper.readValue(s, clazz);
-//      } catch (IOException e) {
-//         e.printStackTrace();
-//      }
-//      return o;
+      //return gsonCompact.fromJson(s, clazz);
```

Comment:
I don't recall exactly but there was a reason I was not using the mapper to convert from JSON to POJO. I think mapper required that all POJO fields exist in the JSON object whereas gson allowed the missing JSON objects leaving the POJO field to null.